### DISCUSSION TITLE
What is the 'try-now-page' module used for?

### DISCUSSION THREAD

**Discussion by wyhdil** [eclipse-theia/theia-cloud#239] [Q&A] [answered]
I have run theia cloud, but I don’t know what the ‘try-now-page’ module does.
![6fbd4c195a8fd950e0a3cfc870d60b99](https://github.com/eclipsesource/theia-cloud/assets/69418800/f7f5d74f-0b31-4757-b305-8a8aa92ee9a4)

And it seems that ‘try-now-page' container was not found in the landing-page pod.
![image](https://github.com/eclipsesource/theia-cloud/assets/69418800/ad4045d0-b9da-469e-9217-0fee50bc54a6)

So I don’t know if this module is used in theiacloud and what its purpose is.

I'm looking forward to your reply, thank you！

**Comment by jfaltermeier** [comment]
You can exchange the landing page via a helm value, which is `landingPage.image`: https://github.com/eclipsesource/theia-cloud-helm/tree/main/charts/theia.cloud
This defaults to `theiacloud/theia-cloud-landing-page:<tag>` but you could use `theiacloud/theia-cloud-try-now-page:tag>`

**Reply by wyhdil** [reply]
In this case, you mean that the purpose of 'try-now-page' is the same as 'landing-page', which is used to log in through keycloak, right?
That being the case, I would like to know what is the difference between these two modules. Thanks!

**Reply by jfaltermeier** [reply, marked as answer]
The landing-page is our default example. It just has the absolutely necessary UI (an icon and a place for messages) to test Theia Cloud. If keycloak is enabled you will be directly redirected, but there are not login or logout buttons. 
We expect that adopters will want to integrate Theia Cloud into their own dashboards/websites, so we don't offer a full blown dashboard ourselves. 

The try-now page on the other hand is actually used in production to host the demo for the Theia IDE on theia-ide.org
This page is maintained by Theia Cloud. We currently have it checked in in the main git repository, but we might move it to a dedicated repository in the future.

**Reply by wyhdil** [reply]
Thank you very much for your reply！！
